result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    if (true) {
+        var build = true;
+    }
+
+    console.log(build);
+}
+
+function doIfElse() {
+    if (true) {
+        var build = true;
+    } else {
+        var build = false;
+    }
+}
+
+function doTryCatch() {
+    try {
+        var build = 1;
+    } catch (e) {
+        var f = build;
+    }
+}
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    var build;
+
+    if (true) {
+        build = true;
+    }
+
+    console.log(build);
+}
+
+function doIfElse() {
+    var build;
+
+    if (true) {
+        build = true;
+    } else {
+        build = false;
+    }
+}
+
+function doTryCatch() {
```

Comment:
__[development][eslint]__
* Error - 'doTryCatch' is already defined (no-redeclare)